cmd/gateway-main_test.go

```diff
@@ -62,23 +62,24 @@ func TestValidateGatewayArguments(t *testing.T) {
 	}
 	nonLoopBackIP := nonLoopBackIPs.ToSlice()[0]
 
+	errMsg := "endpoint points to the local gateway"
```

Comment:
You can re-purpose the message like `errEndpointLoop` ? which is returned by the actual function and you are just using errEndpointLoop.Error() to compare. Naked strings like this can invite issues like typo etc avoids changing at multiple places.